### ISSUE TITLE
With or without using a miner? How to avoid using it?

### ISSUE THREAD

**Issue by samils7** [KevinMusgrave/pytorch-metric-learning#415]
> The high loss value is normal for ArcFace. I think the big spikes downward are caused by the tuple miner. If you get rid of the miner, the loss curve should be smoother.

How can I get rid of miner? I have just tried to pass None into mining_funcs parameter of trainer but the trainer returned with an error.  What should I do in this case? 

I aim to train a classifier with ~350 classes and don't know any idea if I would need to use a miner with ArcFace loss. 

Thanks for the great work on this repo!

**Comment by KevinMusgrave** [comment]
Set it to an empty dictionary: ```mining_func={}```.

**Comment by samils7** [comment]
> Set it to an empty dictionary: `mining_func={}`.

Thank you I have just discovered it. Edited the question.

**Comment by KevinMusgrave** [comment]
If you use a miner with ArcFaceLoss, then each batch element's loss will be weighted by how frequently it appears in the mined pairs/triplets. 

I would leave out the miner, because I don't know if ArcFaceLoss benefits at all from this type of weighting.

**Comment by samils7** [comment]
I misunderstood the concept of the miner. I got it now. Thank you.
